recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
+  version: 2.0.12
+
+source:
+  fn: "uwsgi-2.0.12.tar.gz"
+  url: "https://pypi.python.org/packages/0c/1c/44849e293e367a157f1ad863cee02b4b865840543254d8fae3ecdebdbdb9/uwsgi-2.0.12.tar.gz"
```

Comment:
Yeah, they set this up about 5 days ago. See this [comment]( https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package#comment-27243225 ).